spec/features/user_edits_voter_spec.rb

```diff
@@ -36,7 +36,7 @@
     visit edit_voter_path(voter)
 
     fill_in 'First name', with: ""
-    click_button 'Submit'
+    click_button 'Update Voter'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.